mcts.py

```diff
@@ -25,6 +25,13 @@ def __init__(self, state, parent):
         self.totalReward = 0
         self.children = {}
 
+    def __str__(self):
+        s=[]
+        s.append("totalReward: %s"%(self.totalReward))
+        s.append("numVisits: %d"%(self.numVisits))
+        s.append("isTerminal: %s"%(self.isTerminal))
+        s.append("children: %s"%(self.children.keys()))
```

Comment:
The keys of this children dict are the actions rather than the children nodes, so maybe `possibleActions` would be a better name than `children` in this representation?